src/lib/containers/widgets/SynapseImage.tsx

```diff
@@ -108,7 +108,12 @@ class SynapseImage extends React.Component<
   public render() {
     const { params } = this.props
     const { align = '', altText = 'synapse image' } = params
-    let scale = `${Number(params.scale) ?? 100}%`
+
+    let scale = 'auto'
+    if (params.scale && params.scale !== '100') {
+      scale = `${Number(params.scale)}%`
```

Comment:
Only use the scale parameter if it's set and not '100'.  Default value is 'auto'